pkg/audit/reporter_test.go

```diff
@@ -22,7 +21,8 @@ func TestReportPendingAudits(t *testing.T) {
 	testplanet.Run(t, testplanet.Config{
 		SatelliteCount: 1, StorageNodeCount: 1, UplinkCount: 0,
 	}, func(t *testing.T, ctx *testcontext.Context, planet *testplanet.Planet) {
-		err := planet.Satellites[0].Audit.Service.Close()
+		audits := planet.Satellites[0].Audit.Service
```

Comment:
Because `audits` isn't use for anything else than `Close()` call, I would leave this one as it was.

Reply:
It seems I overused it here. Fixed.